### PR TITLE
Remove .middleware() call

### PR DESCRIPTION
**Pull request by Loskir**
Also move I18n below MyContext so it can be used as I18n<MyContext>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,23 +37,23 @@ Example bot
 import { Bot, Context } from "https://deno.land/x/grammy/mod.ts";
 import { I18n, I18nFlavor } from "https://deno.land/x/grammy_i18n/mod.ts";
 
+// For proper typings and auto-completions in IDEs,
+// extend the `Context` using `I18nFlavor`.
```
Comment (KnorpelSenf) [inline README.md]: Let's do
```suggestion
// customize the `Context` using `I18nFlavor`.
```
while we're at it

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -37,23 +37,23 @@ Example bot
 import { Bot, Context } from "https://deno.land/x/grammy/mod.ts";
 import { I18n, I18nFlavor } from "https://deno.land/x/grammy_i18n/mod.ts";
 
+// For proper typings and auto-completions in IDEs,
+// customise the `Context` using `I18nFlavor`.
```
Comment (KnorpelSenf) [inline README.md]:
```suggestion
// customize the `Context` using `I18nFlavor`.
```

File: README.md [hunk 3]
```diff
@@ -37,23 +37,23 @@ Example bot
...
-// For proper typings and auto-completions in IDEs,
-// extend the `Context` using `I18nFlavor`.
-type MyContext = Context & I18nFlavor;
-
 // Create a bot as usual, but use the modified Context type.
 const bot = new Bot<MyContext>(""); // <- Put your bot token here
 
 // Tell the bot to use the middleware from the instance.
 // Remember to register this middleware before registering
 // other middlewares.
```
Comment (Loskir) [inline README.md]: Other middlewares? User middlewares maybe? Some middlewares (like session) must be installed before

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
sorry, my bad

File: README.md (hunk 3 above)
Comment (KnorpelSenf) [reply README.md]: Maybe “before registering your handlers”?

Comment (Loskir) [reply README.md]: Yep, sounds good

Comment (Loskir) [reply README.md]: Also "Tell the bot to use the middleware from the instance." sounds too specific, doesn't it?

Comment (KnorpelSenf) [reply README.md]: Yes, I agree. It would be enough to say that the i18n plugin gets installed, what do you think?

Comment (Loskir) [reply README.md]: Just remove the first line

Comment (KnorpelSenf) [reply README.md]: Heh, yes that works!

**Review by dcdunkan — APPROVED** [review]

**Review by KnorpelSenf — APPROVED** [review]

**Pull request merged**